client/components/jetpack-header/liquidweb.jsx

```diff
@@ -0,0 +1,160 @@
+/** @format */
```

Comment:
🙈 I'm an SVG nerd…

I'd ask for this to be cleaned up, but I'm resisting.  The SVG is ugly and could be run through an optimizer, but it's not a big deal.